README.md

```diff
@@ -444,15 +438,17 @@ The plugin is currently used on an opt-in basis. To use it, apply the plugin and
 apply plugin: 'com.palantir.baseline-java-versions'
 
 javaVersions {
+    javaCompiler = 25
     libraryTarget = 11
     distributionTarget = 17
     runtime = 21
 }
 ```
 
 The configurable fields of the `javaVersions` extension are:
-* `libraryTarget`: (required) The Java version used for compilation of libraries that are published.
-* `distributionTarget`: (optional) The Java version used for compilation of code used within distributions, but not published externally. Defaults to the `libraryTarget` version.
+* `javaCompiler`: (required) The version of the Java compiler used. 
```

Comment:
should we say that the javaCompiler version can be higher that the target versions because we are using the `--release` capability (cross-compiling capability) ? I am not sure if is too clear for people what all these versions mean/what it affects.